cmd/controller/main.go

```diff
@@ -129,6 +129,7 @@ func main() {
 	k8sInformerFactory := informers.NewSharedInformerFactory(k8sClient, options.ResyncPeriod)
 	pvcInformer := k8sInformerFactory.Core().V1().PersistentVolumeClaims()
 	podInformer := k8sInformerFactory.Core().V1().Pods()
+	configmapInformer := k8sInformerFactory.Core().V1().ConfigMaps()
```

Comment:
Can we restrict this to only operate on the namespaces we care about?

Reply:
Sure. Please see cc165a571de750365bf98ec581bbf0ff9794ffd3.
Thanks!